bin/deploy

```diff
@@ -103,19 +104,34 @@ test -f "$LSSTSW/stack/.deployed" || ( # Adjust the stack config
 test -f "$LSSTSW/versiondb/.deployed" || ( # Clone the version database
     echo "::: Deploying versiondb"
     rm -rf versiondb
-    git clone git@github.com:lsst/versiondb.git
-    (cd versiondb && git config push.default current)
+    if [ -z ${NO_PUSH+x} ]; then
```

Comment:
Isn't this officially spelled `${NO_PUSH:+x}`?  And isn't this backwards?  If NO_PUSH is defined, then the ssh clone and git config are performed.